client/src/Movies/SavedList.js

```diff
@@ -10,9 +13,13 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
```

Comment:
consider using destructuring
```suggestion
        {this.props.list.map(({title, id}) => (
```